registration/scripts/directives.js

```diff
@@ -56,7 +63,13 @@
                             if (angular.isDefined(deferred)) {
                                 deferred.notify('reject', response);
                             }
+                            apiError = true;
                         });
+                    } else {
+                        //check there are no other error messages on the screen from the api
+                        if (!apiError) {
+                            scope.emptyEmail = true;
+                        }
```

Comment:
Furthermore, testing on Humira ePatient with an empty email I get a `400` with `{"email": ["This field is required."]}`